src/config.zig

```diff
@@ -302,7 +302,24 @@ comptime {
     assert(tcp_sndbuf_replica <= 16 * 1024 * 1024);
     assert(tcp_sndbuf_client <= 16 * 1024 * 1024);
 
+    // For the given WAL (lsm_batch_multiple=4):
+    //
+    //   A    B    C    D    E
+    //   |····|····|····|····|
+    //
+    // - ("|" delineates measures/batches; "·" is a prepare in the WAL.)
```

Comment:
```suggestion
    // - ("|" delineates measures, where a measure is a multiple of prepare batches; "·" is a prepare in the WAL.)
```

Reply:
Thanks! I was misunderstanding `lsm_batch_multiple` as "the cardinality of the batch" rather the "the number of batches in a measure".

Reply:
We should find a better name for it!